charts/ucmc-service/Chart.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: v2
 description: A Helm chart for ucmc-service App
 name: ucmc-service
 home: https://github.com/hmcts/ucmc-service
-version: 0.1.0
+version: 0.2.0
```

Comment:
shouldn't it be 0.1.1

Reply:
If we are **not** using semantic versioning here then yes.